src/auth/index.ts

```diff
@@ -4,7 +4,9 @@ import { ContextModule, getContext } from '~context/index.ts';
 import { isFullWallet } from '~types/wallet.ts';
 
 const authProxyRequest = async (urlPartial: string, options?: RequestInit) => {
-  const { host, origin } = window.location;
+  const host = process.env.HOST || 'localhost:9091';
+  const origin = process.env.ORIGIN || 'http://localhost:9091';
+
```

Comment:
Same in this file as well, please remove the fallbacks as they're not needed anymore